nexus/src/app/background/init.rs

```diff
@@ -298,6 +302,20 @@ impl BackgroundTasks {
             )
         };
 
+        let task_v2p_manager = {
+            driver.register(
+                "v2p_manager".to_string(),
+                String::from("manages opte v2p mappings for vpc networking"),
+                // TODO add custom config?
+                // should we create a general setting that can be shared across
+                // multiple tasks? A lot of these have the same values...
```

Comment:
I figured this is worth discussing. We're getting _a lot_ of these config values that ultimately end up having the exact same intervals.